### ISSUE TITLE
Add sub menu to manage account page

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6290]
With the addition of new features, we should add a sub menu to the manage account page, e.g:

![Screenshot from 2019-07-29 07-04-19](https://user-images.githubusercontent.com/3323703/62025198-61e11b80-b1cf-11e9-8843-aa67387a407e.png)

**Comment by di** [comment]
IMO, it might be time to move some or all of these to separate pages as well. This page has gotten quite long and attempting to deep-link to these various sections might not be the best UX.

**Comment by nlhkabu** [comment]
I agree @di, however this is a larger task, so I think a good short term fix is to add the sub menu, with a view of improving this later.

**Comment by nlhkabu** [comment]
Closing in favour of https://github.com/pypi/warehouse/issues/18950
